pkgs/development/compilers/edk2/default.nix

```diff
@@ -65,7 +85,7 @@ edk2 = stdenv.mkDerivation {
 
       buildPhase = ''
         runHook preBuild
-        build -a ${targetArch} -b RELEASE -t GCC5 -p ${projectDscPath} -n $NIX_BUILD_CORES $buildFlags
+        build -a ${targetArch} -b RELEASE -t ${buildType} -p ${projectDscPath} -n $NIX_BUILD_CORES $buildFlags
```

Comment:
Do we need to expose `buildType` here. Could it not just be set based on `stdenv.cc`?

Reply:
True, I moved this up to where `targetArch` is determined.